### ISSUE TITLE
Error when moving jog wheels

### ISSUE THREAD

**Issue by JohannesWiesner** [dan-giddins/mixxx-ddj-200-mapping#3]
I downloaded both the .js file and the .xml file and placed them in the controller folder. Everythings seems to work besides when I am moving one of the jog wheels I am getting the following error:

<img width="600" alt="ddj_200_jog_wheel_error" src="https://user-images.githubusercontent.com/48254290/91661062-10921280-eada-11ea-815d-f31463dbd3c8.png">

which translates to:

"A currently used control is not working properly. The script code must be corrected [...] 
Uncaptured exception on line 1 in passed code: ReferenceError: Can't find variable: DDJ200"

Also in the controller folder I have this file called DDJ-200_0001_2.hid.xml which just appeared without me inserting in in the folder. DDJ-200_0001 is also be shown as a potential controler to choose in the settings. I don't know if this might have something to do with the errror?

**Comment by dan-giddins** [comment]
That's really strange, as line 1 of the passed code (which I am assuming is the .js file) is `var DDJ200 = {};` i.e the variable DDJ200 is being declared [as per the documentation](https://github.com/mixxxdj/mixxx/wiki/midi%20scripting#script-file-header) and therefor shouldn't need to be 'found'...

And yes, the strange 'DDJ-200_0001' and 'DDJ-200_0001_2' files appearing is normal, I have that as well, and I don't know why. It shouldn't be causing this issue, because my .xml file explicitly specifics the .js file, so as long as they are in the same folder and you have selected my mapping, it should just work fine.

I'm not at home right now, so can't test this with the latest version of MIXXX, but the last time I pushed this code I'm pretty sure it was working for me.

All I can recommend is to:
1. Close MIXXX
2. Delete all the files relating to the DDJ-200 controller from the 'controllers' folder
3. Redownload the .js and .xml file into the 'controllers' folder
4. Open MIXXX
5. Make sure the correct .xml mapping is selected in the MIXXX preferences

If that doesn't work, all I can think of is to try downloading an older version of MIXXX if available. The last time I tested this code was about 6 months ago.
